app/uk/gov/hmrc/agentservicesaccount/config/AppConfig.scala

```diff
@@ -149,7 +149,7 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
 
   val agentPermissionsOptInUrl = s"$agentPermissionsFrontendExternalUrl$agentPermissionsFrontendOptInPath"
   val agentPermissionsOptOutUrl = s"$agentPermissionsFrontendExternalUrl$agentPermissionsFrontendOptOutPath"
-  val agentPermissionsCreateAccessGroupUrl = s"$agentPermissionsFrontendExternalUrl$agentPermissionsFrontendGroupsCreatePath"
+  val agentPermissionsCreateAccessGroupUrl: String = "http://localhost:9452/agent-permissions/create-group/select-group-type"
```

Comment:
You will want to keep `$agentPermissionsFrontendExternalUrl` since the URL will not always be localhost:9452

but you can replace the create path instead of updating the `application.conf` agent-permissions-frontend.create-access-group-path = value

eg. you could change this line to s"$agentPermissionsFrontendExternalUrl/agent-permissions/create-group/select-group-type"